### PR TITLE
Create atd-signal-comms DAG

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/12952

## Associated repo
- atd-signal-comms

## Testing

You can spin up local Airflow to see that the DAG parses properly.

Although you can trigger the DAG and it will not fail, it will take a verrry long time to run because it attempts to ping thousands of devices and each ping will timeout and retry.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Add note to 1PW secrets moved to API vault and check for duplicates

### CODE REVIEW

File: dags/atd_signal_comms.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+REQUIRED_SECRETS = {
+    "AWS_ACCESS_KEY_ID": {
+        "opitem": "atd-signal-comms AWS",
+        "opfield": f"production.aws_access_key_id",
+    },
+    "AWS_SECRET_ACCESS_KEY": {
+        "opitem": "atd-signal-comms AWS",
+        "opfield": f"production.aws_secret_access_key",
+    },
```
Comment (johnclary) [inline dags/atd_signal_comms.py:33]: i removed the redundant entries in our developer 1pass vault

**Review by frankhereford — APPROVED** [review]
This parses just like it should on my local stack. I have not tried to run it based on its need for the signals network. Thank you John!

**Comment by johnclary** [comment]
thank you @frankhereford!

**Comment by johnclary** [comment]
i have enabled the production DAG and disabled the v1 DAG ✅

File: dags/atd_signal_comms.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    (
+        start_date
+        >> cameras_s3
+        >> detectors_s3
+        >> dms_s3
+        >> battery_backup_s3
+        >> cameras_socrata
+        >> detectors_socrata
+        >> dms_socrata
+        >> battery_backup_socrata
```
Comment (chiaberry) [inline dags/atd_signal_comms.py:185]: i like this over the old way we'd write tags t1 >> t2 >> etc

Comment (johnclary) [reply dags/atd_signal_comms.py:185]: agreed!

**Pull request merged**